basis/common-router/src/test/java/com/alex_koveckiy/common/router/AbstractByCommandRouterHandlerTest.java

```diff
@@ -0,0 +1,39 @@
+package com.alex_koveckiy.common.router;
+
+import com.alex_koveckiy.common.protocol.ActionHeader;
+import com.alex_koveckiy.common.protocol.Request;
+import com.alex_koveckiy.common.router.api.AbstractByCommandRouterHandler;
+import com.alex_koveckiy.common.router.api.Handler;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.assertThat;
+
+/**
+ * Created by alex on 02.03.17.
+ */
+@RunWith(MockitoJUnitRunner.class)
+public class AbstractByCommandRouterHandlerTest {
+
+    @InjectMocks
+    private AbstractByCommandRouterHandler<Handler> handler =
+            Mockito.mock(AbstractByCommandRouterHandler.class, Mockito.CALLS_REAL_METHODS);
```

Comment:
@Alexey007gold No need to use mock if you use @InjectMocks. Or dont use InjectMocks - you dont have any mocks to inject.